main.go

```diff
@@ -1,16 +1,42 @@
 package main
 
 import (
+	"fmt"
 	"log"
+	"net/http"
 	"os"
+	"time"
 	"trellis-cli/cmd"
+	"trellis-cli/config"
+	"trellis-cli/github"
 	"trellis-cli/trellis"
+	"trellis-cli/update"
 
+	"github.com/fatih/color"
 	"github.com/mitchellh/cli"
 )
 
+const version = "0.9.0"
+
+var updaterEnabled = ""
```

Comment:
```suggestion
var updaterEnabled = "roots/trellis-cli"
```

Or, was it left blank on purpose?

Reply:
Left blank on purpose so that normal builds (just using `go build`) don't include HTTP calls for the update checks.

Oh!! that reminds me I need to update `.gorelease.yml` to use `go build -ldflags="-X main.updaterEnabled=roots/trellis-cli"`

But yeah... I took the name from GitHub's and I agree it should have `repo` in it.